src/dashboards/components/AutoRefresh.scss

```diff
@@ -55,9 +55,20 @@
 
 .timerange-dropdown {
   justify-self: end;
-  width: 192px;
+  width: 178px;
 }
 
 .refresh-form-timerange-toggle {
   margin: 0px $cf-marg-b;
 }
+
+.refresh-input {
+  width: 100px;
+  input {
+    border-radius: 0px 5px 5px 0px;
```

Comment:
use the clockface cf-radius or whatever it is

Reply:
^